conan/cli/printers/list.py

```diff
@@ -0,0 +1,39 @@
+from conan.api.output import Color, cli_out_write
+
+# Keep them so we don't break other commands that import them, but TODO: Remove later
```

Comment:
This breaks those commands. Do we want that? Or should we keep them on list forever?